### ISSUE TITLE
auroracoin github user taken...  ask owner

### ISSUE THREAD

**Issue by SkateFish** [SkateFish/auroracoin-browser-wallet#8]
Hi... i have started to take over management of the auroracoin project.  

would you be willing to give up / sell your github account auroracoin ? 

this is just me doing this as an individual, so please do not try to "get as much as you can get." 

my best offer would be around 2000aur.   

if you are also still a fan or hodler... i can promise you some great things are coming out within the next month

@auroracoin

**Comment by SkateFish** [comment]
@auroracoin

**Comment by auroracoin** [comment]
Hi no auroracoin is just my name on github like john or bill ect. Many links to my name.  Is it that you want that name for auroracoin? I should have chosen a different name. :)  But I don't understand the difference between auroracoin and my profile name Tbear. Is everything linked to @auroracoin or Tbear?

**Comment by SkateFish** [comment]
Yes, I was thinking it would be good for the auroracoin project, of which I
have officially started to manage, to have our new repository (I am
gathering all the various repositories around to place it in one place) as
"auroracoin."

I do not have any access to our developer fund, it is locked up right now.
 However I would be willing to be somewhat generous in giving you from
personal funds.

thanks,
michael hannes

**Comment by auroracoin** [comment]
I'm right in the middle of something right now will look into this asap. BTC would be nice?

**Comment by SkateFish** [comment]
hmmm... if you have many links to your name, how do you think that would
play into re-purposing your auroracoin github handle ?

i suppose it is really just an issue of your commit pull-request history
starting over with what will be your new handle.

in regards to btc, i guess i would be willing to do around .01 btc, i dont
think i am comfortable with spending much more than that.   perhaps i
should get some thoughts from our developer.

michael

**Comment by SkateFish** [comment]
even .01 btc feels a bit much.   but i've already put it out there, so i
hope you are not looking to make a huge windfall with this.

michael

**Comment by auroracoin** [comment]
Really I don't have a commit pull-request history. And I'm getting a procedure done tomorrow so I need a few days to recover. I think I'll do it. :) I'll let you know soon.

**Comment by auroracoin** [comment]
.02 btc and I'll do it... but who do we use as an escrow?  If you accept?
